src/Compression/ICompressionCodec.cpp

```diff
@@ -15,6 +16,13 @@ namespace ErrorCodes
     extern const int CORRUPTED_DATA;
 }
 
+ASTPtr ICompressionCodec::getFullCodecDesc() const
+{
+    std::shared_ptr<ASTFunction> result = std::make_shared<ASTFunction>();
```

Comment:
This code doesn't work and fixed in the consequent PR https://github.com/ClickHouse/ClickHouse/pull/14116. But in this PR it unused.